bootstrap.sh

```diff
@@ -17,7 +17,7 @@ notice "Building Please..."
 go run src/please.go $PLZ_ARGS build //src:please --log_file plz-out/log/bootstrap_build.log
 # Use it to build the rest of the tools that come with it.
 notice "Building the tools..."
-plz-out/bin/src/please $PLZ_ARGS build //package:installed_files --log_file plz-out/log/tools_build.log
+plz-out/bin/github.com/thought-machine/please/src/please $PLZ_ARGS build //package:installed_files --log_file plz-out/log/tools_build.log
```

Comment:
I don't think we should change the output location for binaries. I don't think it particularly fixes much (versus libraries, where it fixes a lot) but it's also going to be quite disruptive (whereas generally users care a lot less about library output locations)